internal/driver/cli.go

```diff
@@ -58,6 +59,7 @@ func parseFlags(o *plugin.Options) (*source, []string, error) {
 	flagTools := flag.String("tools", os.Getenv("PPROF_TOOLS"), "Path for object tool pathnames")
 
 	flagTimeout := flag.Int("timeout", -1, "Timeout in seconds for fetching a profile")
+	flagHTTPPort := flag.Int("http", 0, "Present interactive web based UI at the specified http port")
```

Comment:
I'm OK either way, but I think having http be listed under Output Formats wouldn't be terrible: we have (web|evince|gv) in there already. We could always explicitly disable the command in the UI if we really wanted to (is there really a problem with it?).

In fact, how about calling this "web" instead of "http", and eliminating the current "web"?
There's always "svg" for people that really want a static svg.